Map Version Edits.R

```diff
@@ -16,23 +31,21 @@ library(geosphere)
 library(profvis)
 library(furrr)
 
-townshipa_list <- c("Barrington", "Berwyn", "Bloom", "Bremen", "Calumet", "Cicero", 
-                    "Elk Grove", "Evanston", "Hanover", "Hyde Park", "Jefferson", "Lake", 
-                    "Lake View", "Lemont", "Leyden", "Lyons", "Maine", "New Trier", "Niles",
-                    "North Chicago", "Northfield", "Norwood Park", "Oak Park", "Orland",
-                    "Palatine", "Palos", "Proviso", "Rich", "River Forest", "Riverside", "Rogers Park", 
-                    "Schaumburg", "South Chicago", "Stickney", "Thornton", "West Chicago",
-                    "Wheeling", "Worth")
```

Comment:
This vector wasn't being used for anything, and since it seems identical to the vector of `township_name`s in `ccao::town_dict`, I went ahead and removed it in favor of a comment pointing to the latter township dict.